learn/configuration/instance_options.md

```diff
@@ -50,7 +50,7 @@ Environment variables are always identical to the corresponding command-line opt
 
 **Environment variable**: `MEILI_DB_PATH`
 **CLI option**: `--db-path`
-**Default value**: `"data.ms/"`
+**Default value**: `"data.ms/"`/`"meili_data"` (For Docker)
```

Comment:
It's not really true, the real path is `meili_data/data.ms`
Not sure to understand why we have to specify this for docker.
The default path is still `data.ms`

@irevoire WDYT?

Reply:
Yes the db is in `/meili_data/data.ms` but the default path is still `./data.ms`.
And since the mount point of the docker file has been updated to `/meili_data` you can still access the `data.ms` directly.

Reply:
The changes regarding Docker we did for v0.27.0 are only for our official docker image, it does not impact the Meilisearch instance options
the default values are still `./data.ms` and `/dumps` for the dumps

Reply:
I'm not sure to understand, what would you like to be changed in the docs @juchom?
For me, the current one is already accurate, the section @maryamsulemani97 is not related to the official docker image.